ee/tabby-ui/app/chat/page.tsx

```diff
@@ -136,6 +136,23 @@ export default function ChatPage() {
         document.execCommand('paste')
       } else if ((event.ctrlKey || event.metaKey) && event.code === 'KeyA') {
         document.execCommand('selectAll')
+      } else if (
+        event.key.toLowerCase() === 'l' &&
+        (event.ctrlKey === true || event.metaKey === true)
+      ) {
+        chatRef.current?.focus()
+        event.preventDefault()
```

Comment:
@icycodes this might be the fix for focusing when webview is in focus but chat box is not in focus?